tests/http/clients/ws-data.c

```diff
@@ -118,6 +118,13 @@ static CURLcode recv_binary(CURL *curl, char *exp_data, size_t exp_len)
       fprintf(stderr, "EAGAIN, sleep, try again\n");
 #ifdef _WIN32
       Sleep(100);
+#elif defined (__TANDEM)
+      /* NonStop only defines usleep when building for a threading model */
+# if defined (_PUT_MODEL_) || defined(_KLT_MODEL_)
```

Comment:
```suggestion
#elif defined(__TANDEM)
      /* NonStop only defines usleep when building for a threading model */
# if defined(_PUT_MODEL_) || defined(_KLT_MODEL_)
```
Whitespace nit for consistency.

Reply:
> Or perhaps this, to keeping a single `usleep()` and flat #ifs:
> 
> ```diff
> +#elif defined(__TANDEM) && !(defined(_PUT_MODEL_) || defined(_KLT_MODEL_))
> +      /* NonStop only defines usleep when building for a threading model */
> +      sleep(100);
> ```

I tried that and hit an issue with the `c99` compiler. It did not correctly process the `||`. No way I will get a fix for this in any reasonable time (years). That was in the original attempt, but I had to split it to make it reliable.

Reply:
I wonder how this PP can compile the rest of the code.

Alternatively this may work by avoiding `||`:
```diff
+#elif defined(__TANDEM) && !defined(_PUT_MODEL_) && !defined(_KLT_MODEL_)
+      /* NonStop only defines usleep when building for a threading model */
+      sleep(100);
```